contracts/core/Allo.sol

```diff
@@ -255,6 +255,22 @@ contract Allo is
         _grantRole(pools[_poolId].managerRole, _manager);
     }
 
+    /// @notice Add multiple pool managers
+    /// @dev Emits 'RoleGranted()' event. 'msg.sender' must be a pool admin.
+    /// @param _poolId ID of the pool
+    /// @param _managers The addresses to add
+    function addPoolManagers(uint256 _poolId, address[] memory _managers) external onlyPoolAdmin(_poolId) {
+        for (uint256 i; i < _managers.length;) {
+            if (_managers[i] == address(0)) revert ZERO_ADDRESS();
+
+            _grantRole(pools[_poolId].managerRole, _managers[i]);
+
+            unchecked {
+                ++i;
+            }
+        }
```

Comment:
```suggestion
        for (uint256 i; i < _managers.length;) {
            addPoolManager(_poolId, manager[i]);

            unchecked {
                ++i;
            }
        }
```

I believe we can also make `addPoolManager` internal and remove the `onlyPoolAdmin` modifier since it is being called on `addPoolManagers`